### DISCUSSION TITLE
eligibility problem for zksync airdrop [General] <Title>

### DISCUSSION THREAD

**Discussion by ahooora14** [zkSync-Community-Hub/zksync-developers#578] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

hi dear zk owners, i am one your loyal users who star using your ecosystem from 2 years ago till this moment and im not eligible for the airdrop. its not fair, i spent lot of time to prove you that im loyal and a real fan. its very dissapointing and suprising that im not eligible. here is my wallet address : 0x3a9573eEC4a642C97fED04a0ec3161e06373162b

it will be a nice behavior if you check this out and answer me

### Additional Details

zksync airdrop???

**Comment by 0xsisyfos** [comment, marked as answer]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.
